### DISCUSSION TITLE
Custom config path?

### DISCUSSION THREAD

**Discussion by eminence** [helix-editor/helix#8160] [Q&A] [answered]
Hi all,

Is there an environment variable I can use to tell `hx` to load its config file from a custom path?  Or an environment variable to override the default `~/.config/helix/` config path?

(I know about the `--config` command line option, so I could write a custom wrapper script, but if there's an envvar I could use, that would be ideal).

Thanks!

**Comment by the-mikedavis** [comment, marked as answer]
No only the `--config` CLI argument is supported. You should use a wrapper script as you mention

**Reply by bqback** [reply]
I assume this is still the case? [This comment](https://github.com/helix-editor/helix/issues/5065#issuecomment-1343090009) from you, preceding your answer, seems to note that this was planned, but  your answer points towards this not being implemented. I wouldn't mind this not being configurable if the Windows path for the default locations wasn't ungodly long.
